docs/src/assets/scss/carbon-ads.scss

```diff
@@ -15,7 +15,6 @@
     padding: .6em;
     font-size: 1rem;
     overflow: hidden;
-    border-radius: 4px;
```

Comment:
I remember @nzakas  was telling these dups were in place knowingly. Some browsers did not have css variables support https://caniuse.com/css-variables. so it was intentionally added.

Reply:
OK, I allowed the consecutive duplicate properties, although there are variables used throughout, so I'm not sure if IE11 compat is still even possible. That's the only browser that doesn't support them these days